run_docker.sh

```diff
@@ -1 +0,0 @@
-docker run -ti --rm -v $(pwd):/Qt.py --entrypoint bash mottosso/qt.py27
```

Comment:
Keep this too please. Alternative it typing this line in to test locally, it can be a pain.

Usage

```bash
$ . run_docker.sh
```

The `"."` means to "source" it, as in, "just copy whatever is in this file into the terminal and run it.